internal/trace/metrics.go

```diff
@@ -94,6 +104,20 @@ var (
 		Aggregation: view.Count(),
 		TagKeys:     []tag.Key{keyInstance, keyDialerID, keyErrorCode},
 	}
+	bytesSentView = &view.View{
+		Name:        "alloydbconn/bytes_sent",
+		Measure:     mBytesSent,
+		Description: "The number of bytes sent to an AlloyDB instance",
+		Aggregation: view.Sum(),
```

Comment:
I looked at the CloudSQL Go connector's code, and it used `LastValue`. But I'm thinking that we should use `Sum` instead, because we should be recording the total number of bytes sent/received.

With `LastValue`, the `bytes_sent` and `bytes_received` metrics would only show the most recent number of bytes that were sent/received. Not the accumulated sum. But I need to verify this in Cloud Monitoring.

Reply:
cc @jackwotherspoon I can't remember if we talked about this or not. Sum is probably right, but I might have missed this in your PR review.

Reply:
We also have `count` -- I forget what the difference between `count` and `sum` is here, but either way it should be a cumulative metric.

Reply:
Yes `LastValue` was probably an oversight. I wonder whether `Count` or `Sum` is better... 

I'm leaning towards count that way the monitoring platform (Cloud Monitoring) can decide to sum if it wants or not.

Reply:
Thanks all. I am fine with either `Count` or `Sum`. 

I guess maybe `Sum` would be better if the customer doesn't use Cloud Monitoring, but that seems unlikely

Reply:
Let's use `Sum` for now then if its the most simple

Reply:
I tested this using both Sum() and Count(). I had a client program that does 2 operations: inserts a row into a table and then reads rows from the table. So as this program gets run more times, the number of bytes sent and received should increase over time.

I began using Sum() for both `bytes_sent` and `bytes_received`. And then at different times, I changed between using Count() and Sum(). It looked like this in Cloud Monitoring (this graph is for `bytes_sent`, but `bytes_received` had a similar shaped graph too): 
![cloudmonitoring](https://github.com/user-attachments/assets/ba3f6055-b2dc-4478-a385-84cb4eee0e9f)

I'm thinking that Sum() looks more accurate, because the number of bytes sent is increasing over time.

Reply:
On second thought, I'm thinking that as this program gets run more times, the number of bytes sent will be the same each time that it's run. But the number of bytes received will increase, because the table will grow larger.

The unaggregated graph in Cloud Monitoring might do a better job of showing this. I'll try testing this again

Reply:
With Sum(), I verified that `bytes_sent` is constant: 
![sum_bytes_sent](https://github.com/user-attachments/assets/f2ad211a-1a60-4459-9c4d-b1802fc9fe62). And `bytes_received` increases over time: 
![sum_bytes_received](https://github.com/user-attachments/assets/443d3002-b54a-45ed-bd82-332e8beac369)

Count() has a similar graph, but was showing `bytes_sent` and `bytes_received` as less than 1/s. So I'm guessing this metric is measuring number of requests per unit of time. And not number of bytes per unit time. Here is an example of its `bytes_received` graph: 
![count_bytes_received](https://github.com/user-attachments/assets/ac7b93eb-7ea8-48d7-8d92-0c8496cdd2c8)

LastValue() has units of bytes. Not bytes/sec, unlike Sum(). But I think it's only showing the last number of bytes received from a read query, because it was only showing ~6k bytes received in this example: 
![last_value_bytes_received](https://github.com/user-attachments/assets/cf19727e-2db5-44fc-8e6a-badf25b99ce4)
In the graph above for bytes received for Sum(), it was showing ~68k bytes received over a 3 minute period.

Reply:
Thank you very much for testing all of this!